plexapi/video.py

```diff
@@ -189,12 +189,16 @@ def seasons(self):
         path = '/library/metadata/%s/children' % self.ratingKey
         return utils.listItems(self.server, path, Season.TYPE)
 
-    def season(self, title):
+    def season(self, title=None):
         """Returns a Season
 
         Args:
-            title (str): fx Season1
+            title (str, int): fx Season 1
         """
+        title = str(title)
```

Comment:
I think using isinstance(title, int) might be a better choice here than converting it to a string then checking isdigit()